bigtop-packages/src/charm/hive/layer-hive/reactive/hive.py

```diff
@@ -78,9 +87,13 @@ def install_hive(hadoop):
     else:
         hbserver = None
 
+    # Get zookeepers or None
+    zks = KV.get('zookeepers', None)
```

Comment:
I'm curious why we're using `unitdata.kv` here.  If you look a few lines higher at the hbase block, we could use the same logic to get zks when `is_state(zookeeper.ready)` is available.